packages/core/types/src/http/inventory-level/admin/payloads.ts

```diff
@@ -0,0 +1,16 @@
+export interface AdminUpdateInventoryLevel {
+  stocked_quantity?: number
+  incoming_quantity?: number
+}
+
+export interface AdminCreateInventoryLevel {
+  location_id: string
+  stocked_quantity?: number
+  incoming_quantity?: number
+}
+
+export interface AdminBatchPostInventoryLevelLocation {
+  create?: AdminCreateInventoryLevel[]
+  update?: AdminUpdateInventoryLevel[]
+  delete?: never // TODO - not implemented
```

Comment:
question: It looks like you included this functionality in the PR – is something still missing?

Reply:
Good catch, we didn't have delete param explicitly defined but the action is supported. On the other hand, update params are defined in the schema but update action is not supported in the workflow so I updated the type to reflect that.